frontends/infinite-corridor/src/scss/breakpoints.scss

```diff
@@ -0,0 +1,76 @@
+/*
```

Comment:
This replaces `frontends/infinite-corridor/src/scss/_breakpoints.scss`. This version is a bit different from Open Disucssions, but is consistent with MUI. And Open Discussions had ...a variety ... of breakpoint sets. See PR description.